### DISCUSSION TITLE
Qual o segredo para um home-office funcional e confortável? Vamos falar de setups

### DISCUSSION THREAD

**Discussion by drianoaz** [frontendbr/forum#2504] [Discussão]
Desde o início da pandemia, muitos de nós passamos a trabalhar de casa, e o modelo de home-office se consolidou para muita gente. Começamos então a transformar cantos improvisados em estações de trabalho funcionais. Porém, montar o setup ideal vai muito além de ter um bom computador. Ergonomia, conforto, conectividade e até a organização do espaço podem impactar diretamente na nossa produtividade e bem-estar.

Por aqui, somos duas pessoas dividindo o mesmo escritório em casa. Queremos entender como outras pessoas estão montando seus setups e como estão resolvendo desafios do dia a dia no home-office.

**Aqui estão algumas questões que gostaria de explorar com vocês:**
- 🖥️ **Qual equipamento faz toda a diferença?** Seja teclado mecânico, mouse ergonômico, monitores ajustáveis, ou até uma boa cadeira, o que realmente é essencial para vocês?
- 🎧 **Ambiente e som**: Como vocês lidam com ruídos externos e garantem uma acústica ideal para as reuniões? Usam fones de ouvido com cancelamento de ruído ou apostam em painéis acústicos?
- 💡 **Iluminação**: Natural ou artificial, qual é o segredo para manter o ambiente bem iluminado e agradável? Alguma dica de luminária ou truque para evitar sombras nas videochamadas?
- 🌐 **Conectividade e backups**: Problemas com a internet são um dos maiores vilões do home-office. O que vocês fazem quando a conexão cai? Roteador potente, 4G de backup ou algum plano de contingência? E no caso de oscilações de energia, usam nobreaks ou baterias auxiliares?
- 🪑 **Ergonomia e postura**: Trabalhar sentado o dia inteiro pode ser desafiador. Que tipo de cadeira vocês usam? Alguém já experimentou mesas reguláveis em altura (sit/stand desks)? O que mais ajuda na saúde e bem-estar durante o trabalho?
- 🏡 **Organização do espaço**: Como vocês mantêm o ambiente limpo e organizado para não perder o foco? Usam suportes para monitores, gavetas extras, ou tem dicas para esconder os cabos?
- 📸 **Fotos dos setups**: Quem puder, compartilhe fotos dos seus setups! Ver o espaço de trabalho de outras pessoas pode trazer ideias super úteis para todos nós.

**Bônus**: E quem trabalha em dupla ou divide o espaço com mais alguém? Como organizam o escritório para que todos consigam trabalhar sem se distrair?

**Comment by nao-e-oEdu** [comment]
Não tenho muito o que falar, meu setup é extremamente desconfortável, mas quero acompanhar as respostas desse post. Estou "construindo" o meu setup agr e não faço ideia de onde ou como pesquisar por peças online, quando duvidar da veracidade, e nem de quais são as melhores peças do mercado, além de não querer deixar um assunto desses sem resposta.

**Comment by igorlvicente** [comment]
Usar teclado split e mouse ergonômico fizeram uma diferença enorme na posição dos meus ombros.
Já usei o layout Dvorak há um tempão atrás. Pretendo voltar a usar logo.
Também pretendo aprender bons atalhos pra evitar o uso do mouse sempre que possível.

Quanto à cadeira, uso uma confortável da Cavaletti. Não é das mais caras, mas acho muito boa.

Meu monitor ficava centralizado no meu olho. Sempre achei que olhar pra cima era melhor pra postura, mas entendi que o problema de ficar olhando pra cima era comprimir a parte de trás do pescoço. Hoje, eu mantenho o topo do monitor alinhado com meus olhos.

**Reply by igorlvicente** [reply]
Meu próximo passo é comprar uma mesa ajustável. Não tanto pelo ficar em pé, mas pelo controle de altura. Acredito que poder mudar e adaptar a como estou sentado na hora faz diferença.

Acho que um setup assim (com teclado split e mouse ergonômico) é a meta 😄 
![image](https://github.com/user-attachments/assets/30c456ba-5d52-4b9c-90f4-13e52d73eab7)

**Reply by drianoaz** [reply]
Foi difícil se adaptar para conseguir usar um teclado split?

**Reply by igorlvicente** [reply]
> Foi difícil se adaptar para conseguir usar um teclado split?

Como eu já não "ultrapassava" o limite central do teclado, foi extremamente fácil.
